index.md

```diff
@@ -0,0 +1,222 @@
+
+If you’re a Windows Subsystem for Linux user run the following in your terminal,
+then follow the on-screen instructions to install Wasmtime.
+
+```sh
+curl https://wasmtime.dev/install.sh -sSf | bash
+```
+
+</div>
+
+<div id="platform-instructions-default" class="db">
+
+If you’re running macOS or Linux, download and install Wasmtime by running the
+following in your terminal and following the on-screen instructions.
+
+```sh
+curl https://wasmtime.dev/install.sh -sSf | bash
+```
+
+--------------------------------------------------------------------------------
+
+If you are running Windows, download and run the [Wasmtime
+Installer](https://github.com/bytecodealliance/wasmtime/releases/download/dev/wasmtime-dev-x86_64-windows.msi)
+then follow the on-screen instructions.
+
+--------------------------------------------------------------------------------
+
+Alternatively, you can download releases for all supported platforms from the
+[Wasmtime GitHub page](https://github.com/bytecodealliance/wasmtime/releases).
+
+</div>
+
+</div>
+</section>
+
+<section class="section-tinted">
+<div class="container w-container">
+
+## Example
+
+If you've got the [Rust compiler
+installed](https://www.rust-lang.org/tools/install) then you can take some Rust
+source code:
+
+```rust
+fn main() {
+    println!("Hello, world!");
+}
+```
+
+and compile/run it with:
+
+```sh
+$ rustup target add wasm32-wasi
+$ rustc hello.rs --target wasm32-wasi
+$ wasmtime hello.wasm
+Hello, world!
+```
+
+</div>
+</section>
+
+<section>
+<div class="container w-container">
+
+## Features
+
+* **Fast**. Wasmtime is built on the optimizing [Cranelift] code generator to
+  quickly generate high-quality machine code either at runtime or
+  ahead-of-time. Wasmtime's runtime is also optimized for cases such as
+  efficient instantiation, low-overhead transitions between the embedder and
+  wasm, and scalability of concurrent instances.
+
+* **[Secure]**. Wasmtime's development is strongly focused on the correctness of
+  its implementation with 24/7 fuzzing donated by [Google's OSS Fuzz],
+  leveraging Rust's API and runtime safety guarantees, careful design of
+  features and APIs through an [RFC process], a [security policy] in place
+  for when things go wrong, and a [release policy] for patching older versions
+  as well. We follow best practices for defense-in-depth and known
+  protections and mitigations for issues like Spectre. Finally, we're working
+  to push the state-of-the-art by collaborating with academic
+  researchers to formally verify critical parts of Wasmtime and Cranelift.
+
+* **[Configurable]**. Wastime supports a rich set of APIs and build time
+  configuration to provide many options such as further means of restricting
+  WebAssembly beyond its basic guarantees such as its CPU and Memory
+  consumption. Wasmtime also runs in tiny environments all the way up to massive
+  servers with many concurrent instances.
```

Comment:
The two "such as"s in the first sentence makes it a bit of a journey. Maybe something like:

```suggestion
* **[Configurable]**. Wasmtime uses sensible defaults, but can also be
  configured to provide more fine-graned control over things like CPU
  and memory consumption. Whether you want to run Wasmtime in a tiny
  environment or on massive servers with many concurrent instances,
  we've got you covered.
```